hub/powertoys/grouppolicy.md

```diff
@@ -90,49 +251,145 @@ This policy configures whether the action center notification for new updates is
+- Type: DWORD
+- Example value: `0x00000000`
+
+##### Intune information
+
+- OMA-URI: ./Device/Vendor/MSFT/Policy/Config/PowerToys~Policy~PowerToys~PowerToysRun/PowerToysRunAllPluginsEnabledState
+- Example value: `<disabled/>`
+
 #### Configure enabled state for individual plugins
 
-> [!NOTE]
-> This policy is supported on PowerToys 0.75.0 and later.
+> Supported on PowerToys 0.75.0 or later.
 
-With this policy you can configures an individual enabled state for each PowerToys Run plugin that you add to the list.
+With this policy you can configure an individual enabled state for each PowerToys Run plugin that you add to the list.
 
 If you enable this setting, you can define the list of plugins and their enabled states:
 
-- The value name (first column) is the plugin ID. You will find it in the plugin.json which is located in the plugin folder.
+- The value name (first column) is the plugin ID. You will find it in the `plugin.json` file which is located in the plugin folder.
 - The value (second column) is a numeric value: 0 for disabled, 1 for enabled and 2 for user takes control.
 - Example to disable the Program plugin: `791FC278BA414111B8D1886DFE447410 | 0`
 
 If you disable or don't configure this policy, either the user or the policy "Configure enabled state for all plugins" takes control over the enabled state of the plugins.
 
-You can set the enabled state for all plugins not listed here using the policy "Configure enabled state for all plugins".
+You can set the enabled state for all plugins not controlled by this policy using the policy "Configure enabled state for all plugins".
 
 > [!NOTE]
 > Changes require a restart of PowerToys Run.
+
+##### Group Policy (ADMX) info
+
+- GP unique name: PowerToysRunIndividualPluginEnabledState
+- GP name: Configure enabled state for individual plugins
+- GP path: Administrative Templates/Microsoft PowerToys/PowerToys Run
+- GP scope: Computer and user
+- ADMX file name: PowerToys.admx
+
+##### Registry information
+
+- Path: Software\Policies\PowerToys\PowerLauncherIndividualPluginEnabledList
+- Name: `<PluginID>`
+- Type: STRING
+
+- Example value:
+
+```text
+Software\Policies\PowerToys\0778F0C264114FEC8A3DF59447CF0A74 = 2 (=> User can enable/disable the OneNote plugin.)
+Software\Policies\PowerToys\791FC278BA414111B8D1886DFE447410 = 0 (=> Program plugin force disabled.)
+Software\Policies\PowerToys\CEA0FDFC6D3B4085823D60DC76F28855 = 1 (=> Calculator plugin force enabled.)
+```
+
+##### Intune information
+
+- OMA-URI: ./Device/Vendor/MSFT/Policy/Config/PowerToys~Policy~PowerToys~PowerToysRun/PowerToysRunIndividualPluginEnabledState
+
+- Example value:
+
+> [!NOTE]
+> Syntax: `<PluginID>&#xF000;<Number>&#xF000;<PluginID>&#xF000;<Number>`
+
+```text
+<enabled/>
+<data id="PowerToysRunIndividualPluginEnabledList" value="0778F0C264114FEC8A3DF59447CF0A74&#xF000;2&#xF000;791FC278BA414111B8D1886DFE447410&#xF000;0&#xF000;CEA0FDFC6D3B4085823D60DC76F28855&#xF000;1"/>
+```
```

Comment:
Need to improve layout here. https://learn.microsoft.com/en-us/contribute/content/markdown-reference